SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -32,12 +32,19 @@ const FString SchemaCompilerExe = CreateExePath(GDKProgramPath, TEXT("schema_com
 const FString SpatialOSDirectory = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectDir(), TEXT("/../spatial/")));
 const FString SpatialOSConfigFileName = TEXT("spatialos.json");
 const FString ChinaEnvironmentArgument = TEXT(" --environment=cn-production");
+const FString PlatformVersion = TEXT("x86_64-win32");
+const FString RuntimeExe = TEXT("runtime.exe");
+const FString InspectorExe = TEXT("inspector.exe");
+
+// Timeout given in seconds.
+const int32 ProcessTimeoutTime = 60;
```

Comment:
To be honest I would prefer this constant to be inside SpatialCommandUtils. I can't imagine us using this value any where else.